keps/2724-topology-aware-scheduling/README.md

```diff
@@ -163,6 +165,11 @@ consecutive ranks to be placed within the same topology domain (if possible).
 
 Extension: support for indicating the order of pods for external Jobs.
 
+#### Story 5
```

Comment:
@mimowo @tenzen-y 

Should we include LWS for "two-level" scheduling?

I think there will similar work for JobSet/LWS.

Reply:
yes, extending to "LWS" is definetly on our radar. However, for LWS we need to co-schedule PodSets rather than "chunk" them. So, we consider it a follow up.